models/topic.model.js

```diff
@@ -20,12 +20,24 @@ exports.retrieveArticleById = (article_id) => {
     });
 };
 
-exports.retrieveArticles = () => {
+exports.retrieveArticles = (topic) => {
+  const queryValues = [];
+  let queryStr = `
+  SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url 
+  FROM articles 
+  JOIN comments 
+  ON articles.article_id = comments.article_id`
+  
+  if (topic) {
+    queryValues.push(topic);
```

Comment:
what if the topic is 'DROP TABLE;'? I think you need to check for the available topics in the controller to make sure that your topic is valid :)